code/modules/mob/living/simple_animal/hostile/zombie.dm

```diff
@@ -32,11 +32,13 @@
 /mob/living/simple_animal/hostile/zombie/proc/setup_visuals()
 	var/datum/job/job = SSjob.GetJob(zombiejob)
 
-	var/datum/outfit/outfit = new job.outfit
-	outfit.l_hand = null
-	outfit.r_hand = null
+	var/datum/outfit/outfit = job.outfits["Default"]?[SPECIES_HUMAN]
+	//outfit.l_hand = null
+	//outfit.r_hand = null
 
 	var/mob/living/carbon/human/dummy/dummy = new
+	dummy.temporarilyRemoveItemFromInventory(dummy.get_item_for_held_index(1))
+	dummy.temporarilyRemoveItemFromInventory(dummy.get_item_for_held_index(2))
```

Comment:
What was this changed for

Reply:
Making a new outfit by picking from the list wasn't working for me, for whatever reason. I just made it use the existing outfit and remove the items from the dummy instead of making a new outfit and modifying it. If there's a smarter way to handle this, let me know.

Reply:
qdel em

Reply:
done